app/components/ahoy_captain/comparison_link_component.rb

```diff
@@ -22,12 +26,12 @@ def render?
 
   def options_for_option
     [
-      (link_to "Disable Comparison", AhoyCaptain::Engine.routes.url_helpers.root_path(**helpers.search_params.merge(comparison: false))),
-      (link_to "Previous period", AhoyCaptain::Engine.routes.url_helpers.root_path(**helpers.search_params.merge(comparison: :previous)), class: selected(:previous, :true)),
-      (link_to "Year-over-year", AhoyCaptain::Engine.routes.url_helpers.root_path(**helpers.search_params.merge(comparison: :year)), class: selected(:year)),
       (link_to "Custom period", "javascript:customComparisonModal.showModal()", class: selected(:custom)),
+      (link_to "Year-over-year", AhoyCaptain::Engine.routes.url_helpers.root_path(**helpers.search_params.merge(comparison: :year)), class: selected(:year)),
```

Comment:
if you're already updating, change hardcoded strings to an `i18n` lookup, e.g., "Year-over-year" -> ".year-over-year"